docs/en/02_Developer_Guides/08_Performance/02_HTTP_Cache_Headers.md

```diff
@@ -14,40 +132,39 @@ headers:
   * Since a visitor cookie is set, the site won't be cached by proxies.
   * Ajax requests are never cached.
 
-## Customizing Cache Headers
+## Max Age
+
+The cache age determines the lifetime of your cache, in seconds.
+It only takes effect if you instruct the cache control
+that your response is public in the first place (`setPublic()`).
```

Comment:
I think we need to rely less on this behaviour as there is a slight difference between a cache control header with and without `public`.

Really, the way we'd enable caching by default is the same way we'd do it currently (before this PR was written) and that is to remove the defaults from `HTTP.cache_control` config.

Reply:
Changed to use `enableCache()` instead, and mention "modifying the vary header" as another potential way to ensure cacheability.